processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -19,15 +19,16 @@
 package org.mapstruct.ap.internal.conversion;
 
```

Comment:
@agudian There is also `IMPORT` maybe this one is the one that makes sure there is only one line.